bugsnag/celery/__init__.py

```diff
@@ -28,4 +29,5 @@ def connect_failure_handler():
     Connect the bugsnag failure_handler to the Celery
     task_failure signal
     """
+    bugsnag.configure().runtime_versions['celery'] = celery.__version__
```

Comment:
I think this change is missing test coverage.

Reply:
It is, so the change was manually tested using the example. I've raised an internal ticket to consider adding unit tests to this integration.